src/routers/admin/admin.py

```diff
@@ -1,15 +1,17 @@
 from fastapi import APIRouter
```

Comment:
```suggestion
from fastapi import APIRouter, Header
from typing import Optional
```
인증 방식을 미들웨어나 데코레이터로 돌릴거에요?
아니면 일단은 지금 만들어놓은 인증 함수 사용할거에요?